.github/workflows/demo/docker-compose.demo.yml

```diff
@@ -0,0 +1,10 @@
+version: "3"
+services:
+  haystack-api:
+    restart: always
+  elasticsearch:
+    restart: always
+  ui:
+    restart: always
+    environment:
+      DISABLE_FILE_UPLOAD: 1
```

Comment:
I will add a couple more env vars in this file before we merge this PR

Reply:
Hm It's harder than I though to add stuff to your PR :sweat_smile: I'll wait for the merge and do it afterwards, no problem